packages/kolibri-tools/lib/i18n/utils.js

```diff
@@ -16,8 +16,13 @@ function writeSourceToFile(filePath, fileSource) {
 
 // Compile all of the defined strings & context from the CSVs that have been downloaded
 // from Crowdin.
-function parseCSVDefinitions(dir, subDir = '**') {
-  return glob.sync(path.join(dir, 'CSV_FILES', subDir, '*.csv')).reduce((acc, filePath) => {
+function parseCSVDefinitions(dir, intlLangCode) {
```

Comment:
if `intlLangCode` is optional, maybe set default to `null`

```suggestion
function parseCSVDefinitions(dir, intlLangCode = null) {
```